src/colvarproxy.cpp

```diff
@@ -613,6 +609,21 @@ int colvarproxy::post_run()
 }
 
 
+void colvarproxy::set_total_forces_invalid()
+{
+  std::fill(atoms_total_forces.begin(), atoms_total_forces.end(), cvm::rvector(0.0, 0.0, 0.0));
+  std::fill(atom_groups_total_forces.begin(), atom_groups_total_forces.end(),
```

Comment:
I'm not sure if it's necessary to clear the buffers. It looks fine even if the GPU buffers are not cleared.

Reply:
Either clearing or not is OK. I believe `set_total_forces_invalid` is not called very frequently so the performance is an issue.